### ISSUE TITLE
define ctspecimen_event_type

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#9782]
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type 

>event type

is not useful and needs to be.

https://handbook.arctosdb.org/documentation/specimen-event.html isn't terribly helpful.

I want to say something about "major happening in the life/existence of the object/concept represented by the catalog record" but I don't think that quite aligns with the data.

* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type#depiction feels like a misplaced https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcitation_type_status#basis_of_illustration (and has no other documentation)
* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type#not_detected isn't "major" in any way (and has been used once and therefore should probably be removed)
* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type#observation maybe isn't much more significant to the observed, but I do understand the purpose served (an "Occurrence" was noted without evidence - which suggests perhaps a partless https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type#encounter)
* https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type#use I think is intended for about the shape I see in my head (https://arctos.database.museum/guid/UAM:EH:UA2020-008-0001?seid=4922523), but the definition doesn't quite agree with that and would allow nearly anything.
* https://github.com/ArctosDB/arctos/issues/9781 would ***remove*** major events from this concept/node (and should probably be hard-blocked by this, I don't think we can know where that belongs without knowing what we're doing here)

I'm not sure if we need to adjust the definition/shape of the concept, the data, both, the structure, ?????

HELP!

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by Jegelewicz** [comment]
From the Handbook

> Event Type
> 
> The action during the [Event](https://handbook.arctosdb.org/documentation/collecting-event.html) that affected the [Record](https://handbook.arctosdb.org/documentation/catalog.html).
> 

That seems like a definition. These terms should be action verbs, other than that should there be any constraints?

**Comment by dustymc** [comment]
>should there be any constraints

There's definitely a pure event-based model way out that way, but I don't think we're ready to move 'determined sex' and 'subsampled part' into events. If the answer to "what's an event?" isn't "everything" then we need to know what belongs here (vs. some more specialized structure, like parts or record attributes) in order to not do arbitrary things, which inevitably prohibit systematic discovery/research.

**Comment by Jegelewicz** [comment]
It seems like the question we are asking is - when do I use an event and when do I use an attribute? And I think the answer has to do with whether the location and date of the action are important. If it matters WHEN and/or WHERE something happened, then an event is sorta required because there is no place for that information in an attribute.

**Comment by dustymc** [comment]
> when do I use an event and when do I use an attribute? 

That is exactly what lead me here (but there are probably lots more similar questions that we might get in front of).

>matters WHEN and/or WHERE

... (plus attributes and such) and - yep, I think that's the meat of it.

(And so https://github.com/ArctosDB/arctos/issues/9781 should probably NOT be completed, because the when-and-where-and-metadata - most basic probably being the very poorly named https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcollecting_source - are likely critical to lots of scenarios/research - yes??)

**Comment by campmlc** [comment]
All the discussion above fails to account for mutiple events over time for the same animal. I am requesting attributes because we do not (cannot) put all the events for an animal that lives for many years (elephant?) in one record. We have zoo animals that have blood and serum samples taken multiple times a year over many years or decades. Each collecting event is cataloged as a separate record. We have tried an event based model for these (Mexican wolves) and have determined through many failed attempts over the years that we cannot use events in this contex, not only because of multiple data issues (for example,  record can only have one accession but these involve multiple accessions), but also because of data entry workflows and permission levels required for operators.  I would be happy to go into that in a separate discussion.

In records from the same animal over different events, age and numeric age are entered as attributes for each record. These values change over time, but calculating them depends on knowing the birth date and death date. Currently we record this information in free text in the remarks of these attributes. Putting all these data in together in attributes ensures that all the relevant data are in one place and searchable in one place through the same method. 

From a data entry perspective, we use the "PULL" function in single record entry to tie new records to previous records from the same animal, with a "same individual as" relationship.  Because these are different dates and sometimes localities (animals move between zoos), we do not pull the events in data entry. However, we can pull attributes., ensuring that the same data follow the same animal between records without requiring re-transcription.
These are the reasonsfor the request in https://github.com/ArctosDB/arctos/issues/9781.

**Comment by Jegelewicz** [comment]
Once these dates are known, it seems to me that all you would need are a catalog record for birth and a record for death, each with a single event of an appropriate type. Combining these with all the "collection", "encounter" and other type records in an Arctos Entity would give a complete series of events over the course of the organism's life.
